lib/pka_dev.c

```diff
@@ -662,6 +670,14 @@ static int pka_dev_create_shim(pka_dev_shim_t *shim, uint32_t shim_id,
     shim->shim_id    = shim_id;
     shim->base       = shim_base;
     shim->size       = shim_size;
+    shim->eip154_mmio_base  = shim_eip154_mmio_base;

```

Comment:
I suggest using a separate data structure to capture the resource info.

Reply:
Ack, thanks.